packages/iframe-wallet-poc/src/iframe/user.ts

```diff
@@ -152,7 +152,10 @@ export class User implements machine.mixins.Observable, cf.node.ResponseSink {
   }
 
   public generateObserverNotification(notification: cf.node.Notification): any {
-    return notification.data.results.find(result => result.opCode === 0).value;
+    const PROPOSE = machine.instructions.Opcode.STATE_TRANSITION_PROPOSE;
+    return notification.data.results.find(result => {
```

Comment:
@IIIIllllIIIIllllIIIIllllIIIIllllIIIIll There's no need to expand the arrow function, we can still do:

```ts
return notification.data.results.find(
  result => result.opCode === PROPOSE
).value;
```

Reply:
yeah an old linter rejected that...